scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -628,3 +628,34 @@ <h4 class="modal-title" id="filterNameModalLabel">Filter display name</h4>
   {% endif %}
 
 {% endmacro %}
+
+{% macro pagination_hidden_div(page) %}
```

Comment:
I find it a bit confusing to have 2 macros to create a simple footer with just 2 buttons. Wouldn't it be more readable with just one? You could always put the submits in pagination_footer (where they'd be close to their labels), and pass it also the number of page..

Reply:
Sorry, I don't quite follow you?
The beauty of this version - thanks to an anonymous forum participant - is that it keeps the inputs hidden within the form, while the labels act as proxy outside. And it still is in keeping with the html definition, and works with IE. If I were to, say, move the inputs to the footer without they would just stop working due to the introduction of new forms things like linking to IGV directly or dismissing variants. And if I make new submits outside the main filter form with `form="filters_form"` that will only work for some modern browsers.. 

I still think it may be possible to do this with java script, though I struggled to do it without importing some jQuery helpers, but then I would still need at least two macros - one for the buttons, and one for the js code?

Reply:
Oh, and I perhaps forgot to mention: the STR page link solution will not work on the larger filters. As soon as you open for things like arbitrary gene lists, GET is not really feasible and one has to go with POST:

Reply:
Your solution works and it's fine by me, but please add a couple of comments in the 2 macros referencing the other one, mostly for future maintenance. Took me a while to understand the situation there! :)

To use a GET request you what you could do is:
- Save the filter query in a session variable, whenever the user sends a POST request to the variants view.
- Use this variable whenever the variants view is accessed by GET request
- Have a single macro with the pagination, and making it cooler even: https://getbootstrap.com/docs/4.0/components/pagination/
- This pagination can send GET requests to the variants view with page= as argument.
- You can create a paginator with a custom number pages because you know already:
  - The number of variants returned by the query:
![image](https://user-images.githubusercontent.com/28093618/91528961-b726b980-e908-11ea-9645-0ee4c745b52c.png)
  - The number of variants in each page.

But I understand this is more complicated and our main concern is to fix the broken links now, so you choose!